plugins/python-build/share/python-build/pyston-0.6.1

```diff
@@ -0,0 +1,23 @@
+case "$(pyston_architecture 2>/dev/null || true)" in
+"linux64" )
+  install_package "pyston-0.6.1-linux64" "https://github.com/dropbox/pyston/releases/download/v0.6.1/pyston-0.6.1-linux64.tar.gz#b09516d8e421cb004ef39d832bae600face4ecf80d9791469fb2c026bcf1faae" "pyston" verify_py27
```

Comment:
It seems pyston's binary is linked against `libreadline6` and `libssl1.0.0`. I couldn't run it on my Debian GNU/Linux sid box. It might be better to add `require_distro` to show warnings on unsupported distro versions.

https://github.com/pyenv/pyenv/blob/d1ff898a2bfbe44d74bf92c2b6c9f7652b1d3aaa/plugins/python-build/share/python-build/pypy2-5.6.0#L3